postgres-free-tier.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'The economics of a Postgres free tier'
+description: "Let's look at the numbers behind our free tier. How much does a database costs us and why are we giving them for free."
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-direct-access.jpg
+  alt: Postgres Platform Free Tier
+author: Tudor Golubenco
+date: 07-11-2024
+tags: ['engineering']
+published: true
+slug: postgres-free-tier
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-direct-access.jpg
+---
+
+Our free tier doesn't pause databases after a few days, nor does it have cold starts. A database that you might have started two years ago and never touched since will still be there, upgraded to the latest version, instantly serving your queries.
+
+With high-availability included and fully managed and monitored by us, you can count on it just like you count on an online Google spreadsheet or any other SaaS product. But it is a powerful Postgres database, and comes with 15 GB of storage for free.
+
+When we tell people about this, we're often met with disbelief, quite understandably. So let's see how we do it and if it's going to bankrupt us.
+
+## Why even have a free tier?
+
+To state the obvious, the goal of a free tier is to ultimately generate paying customers. The cost of the free tier is mostly justifiable as part of the cost of finding new customers, similar to e.g. marketing expenses.
+
+A typical funnel looks something like this:
+
+![Free tier funnel](images/freetier-funnel.png)
+
+At the high level, from the new users that enter the funnel, some become inactive soon after. That can be because the product didn't meet their needs or that they were simply looking around without an actual project to work on. In order for a free tier to be financially reasonable, given enough databases, it's essential that the cost per "inactive database" is minimal.
```

Comment:
```suggestion
At a high level, from the new users that enter the funnel, some become inactive soon after. That can be because the product didn't meet their needs or that they were simply looking around without an actual project to work on. In order for a free tier to be financially reasonable, given enough databases, it's essential that the cost per "inactive database" is minimal.
```